examples/llm_inference.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+DTYPE=${DTYPE:-float16}
+MODEL_PATH=${MODEL_PATH:-/home/jundu/dujun/LLM_ckpt/llama2}
```

Comment:
using a general device folder name

Reply:
Got it, has renamed to -/home/username/model_path